cjs/src/server.js

```diff
@@ -1,16 +1,8 @@
-const http = require('http');
-const app = require('./app');
-const chalk = require('chalk');
+const mongooseConnect = require('./db.connect');
+const { app } = require('./app');
 
 const port = process.env.PORT || 3000;
-const server = http.createServer(app);
 
-/* eslint-disable no-console */
-
-server.listen(port, () => {
-  try {
-    console.log( chalk.cyanBright(`\nServer running at http://localhost:${port}`) );
-  }catch(err) {
-    console.log(err);
-  }
+app.listen(port, () => {
+  mongooseConnect(port);
 });
```

Comment:
Thank you @samuko-things 
Approving and sending over to a cleanup branch for further cleanup and testing